README.md

```diff
@@ -50,6 +50,7 @@ Activate the module by adding it to the config.js file as shown in the example b
 | odometer        | Total mileage of the vehicle (rounded to 2 decimal places)                                  |
 | temperature     | Temperature outside and inside the vehicle (see note below)                                 |
 | data-time       | How long ago the data was collected by TeslaFi                                              |
+| location        | Displays current location as tagged in TeslaFi - Is hidden if driving or vehicle's location is not tagged |
```

Comment:
Could you change the wording here please? Doesn't completely make sense to me!

Perhaps - "Displays the current location as tagged in TeslaFi. Not shown if the vehicle is driving, or if the current location isn't tagged"? What do you think?

Might be a good idea to include a link to https://teslafi.com/locations.php with that as well?